user/articles/120-search/04-sort-widget.markdown

```diff
@@ -0,0 +1,147 @@
+permissions can navigate to Control Panel &rarr; Configuration &rarr; Search.
+From there, open the Field Mappings tab and browse the mappings for each index.
+Scroll to the `properties` section of the mapping, and find any `keyword` field,
+`date` field, or a field with any numeric datatype. The `type` field is
+instructive:
+ 
+    "type" : "keyword"
+
+    "type" : "date"
+
+    "type" : "long"
+
+What if you really need to sort by a `text` field? You can do it by adding a new
+version of the field to the index, with the type `keyword`. Don't worry, you
+won't need to code anything to do this. From the field mappings screen mentioned
+above, look at the `firstName` field in the index called `liferay-[companyID]`.
+In fact, look at the next entry as well:
+
+```json
+"firstName" : {
+    "type" : "text",
+    "store" : true
+},
+"firstName_sortable" : {
+    "type" : "keyword",
+    "store" : true
+},
+```
+
+There's a corresponding field with the suffix `_sortable`, and of the correct
+type for sorting (`keyword`). How did that get there? Via the [portal
+property](https://docs.liferay.com/portal/7.2-latest/propertiesdoc/portal.properties.html#Lucene%20Search) 
+
+```property
+index.sortable.text.fields=firstName,jobTitle,lastName,name,screenName,title
+```
+
+All the text fields listed here will have a `fieldName_sortable` counterpart
+created automatically in the index. To add more, copy this value into a
+`portal-ext.properties` file into your Liferay Home folder, add any new field
+names you need to sort by, and restart the server.
+
+#### Adding New Sort Options
+
+To sort by the new field, use the plus symbol below any option's _Field_
+configuration make sure to use the `fieldName_sortable` version of the field in
+the widget configuration. 
+
+To add a new sort option that's already of the proper datatype, use the plus
+symbol below any option's _Field_ configuration and fill in the fields. The
+order of options here in the configuration screen matches the order Users see in
+the select list while configuring the widget for their search.
+
+#### Editing and Deleting Sort Options
+
+To edit an existing option, edit the text in its configuration section.
+
+To delete an existing option, use the minus symbol below its _Field_
+configuration.
+
+### Controlling the Sort Order
+
+To control the order for the sort option, add a plus or minus symbol after the
+`fieldName`. Look how it's done for the existing options called _Created_ and
+_Created (oldest first)_ to understand how it works:
+
+**Label:** _Created_
+**Field:** `createDate-`
+
+The `-` sign following the field name indicates that the order is _descending_.
+Choosing to sort with this will bring search results created most recently to
+the top of the list.
+
+**Label:** _Created (oldest first)_
+**Field:** `createDate+`
+
+The `+` sign following the field name indicates that the order is _descending_.
```

Comment:
Change: "descending" appears twice: 

You can indicate the sort order by appending a + symbol (means, ascending) or a - symbol (means, descending) to the field name like createDate+